pages/_app.tsx

```diff
@@ -61,9 +64,19 @@ function MyApp({ Component, pageProps }: AppProps) {
   return (
     <>
       <Head>
-        <title>Railgun Shielder App</title>
-        <meta name="description" content="An app to shield funds into railgun" />
-        <link rel="icon" href="/favicon.ico" />
+        <title>{APP_TITLE}</title>
+        <meta name="description" content={APP_DESCRIPTION} />
+        <link rel="icon" href="protect-icon.svg" />
+
+        <meta property="og:title" content={APP_TITLE} />
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="https://tokenshielder.com" />
+        <meta
+          property="og:image"
+          content="https://raw.githubusercontent.com/ScopeLift/railgun-shielder/9d8b5dda5826c8121a9934da484a29645de161e6/public/home.png"
```

Comment:
You can't point to local files here so I pushed a commit that uploaded the file, got its permalink, then pushed another commit. More info here: https://stackoverflow.com/a/74917023

Another option is we can convert the png file to base64 and inline it here, or the alternatives discussed here, but this approach seems fine. Also note that the image is a bit chunky at 1.3 MB but it's not part of the actual bundle so that seems ok?